src/test/test_sweep_table_view.py

```diff
@@ -0,0 +1,71 @@
+from unittest import mock
+
+import pytest
+
+
+from PyQt5.QtCore import QModelIndex, Qt, QPoint
+from PyQt5 import QtGui
+
+from sweep_table_view import SweepTableView
+from sweep_table_model import SweepTableModel
+from sweep_plotter import SweepPlotConfig
+from main import SweepPage
+
+from .conftest import mock_called_with, mock_not_called
+
+
+class MockPlotter:
+
+    def __init__(self, full):
+        self._full = full
+
+    def full(self):
+        return self._full
+
+
+@pytest.mark.parametrize("row", [0, 1, 2])
+@pytest.mark.parametrize("col", [ii for ii in range(8)])
```

Comment:
just curious, is there a reason you used list comprehension instead of just `list(range(8))`

Reply:
not really. They seem equivalently readable and the performance is not an issue with 8 elements.  It seems like you disagree about the readability, so I will change it.

Reply:
haha sorry I might be too picky

Reply:
not at all!